tests/AddPatient.test.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable prettier/prettier */
+ 
+const { test } = require('playwright/test');
+
+let page;
+
+test.beforeAll(async ({ browser }) => {
+  page = await browser.newPage();
+});
+ 
+test.afterAll(async () => {
+  await page.close();
+});
+ 
+const AddNewPatientPage  = require('../playwright/pages/AddNewPatientPage');
+// eslint-disable-next-line import/no-unresolved
+
+// This below test for adding new user
+
+// eslint-disable-next-line no-empty-pattern
+test('AddNewPatient test', async ({}, testInfo) => {
+     
+    // eslint-disable-next-line prettier/prettier
+     
+    const addNewPatientPage = new AddNewPatientPage(page);
+    await addNewPatientPage.AddPatient();
+    const screenshot = await page.screenshot();
```

Comment:
Why are we adding a screenshot here?

Reply:
this will add to the test report

![image](https://github.com/wc-phdc-hia/testing/assets/5382920/b5f143bf-5c46-41c5-b5cb-0850d3ff3e5e)